FZJ-IEK3-VSA/HiSim
# Hplib testing

- fix pytest cluster 
- fix problem 1: grid consumption of hp_with_cooling < hp_without_cooling -> solution: changes on hp controller: use higher running and idle times instead of offet temperature because this caused the difference in the hp controller modes
- fix problem 2: EMS hard to understand and seems buggy -> solution: debug, test and tidy up EMS. See grid consumptions of all components now
- see results from ems test here: https://drive.google.com/drive/folders/1sC3eo4W6gtxXToYZk6ABgUp_0ZwEb0zu?usp=sharing
- fix rooftop area scaling
- make household_heat_pump and household_gas_heater runable on cluster
